backend/app/services/project_service.py

```
@@ -422,7 +450,85 @@ def _get_project_tasks(db: Session, project_id: int) -> list[ProjectTaskResponse
     return result
+
+
+def _get_active_project(db: Session, project_id: int, user_id: int) -> Project:
+    """Return an active project owned by a user."""
+
+    project = (
+        db.query(Project)
+        .filter(
+            Project.id == project_id,
+            Project.user_id == user_id,
+            Project.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Project.is_active:` for truth checks

Replace with `Project.is_active`

(E712)